packages/contracts-bedrock/deploy-config/hardhat.json

```diff
@@ -62,5 +62,6 @@
   "daChallengeWindow": 100,
   "daResolveWindow": 100,
   "daBondSize": 1000,
-  "daResolverRefundPercentage": 50
+  "daResolverRefundPercentage": 50,
+  "l2ProxyAdminOwnerVerification": "equal"
```

Comment:
What do you think about changing this to something it's easier to understand, like "productionAddresses: true/false". I don't see a case where no-check would be needed.